docs/doc/10-deploy/05-deploying-local.md

```diff
@@ -135,9 +137,17 @@ The following steps will guide you through the process of locally deploying Data
 
 ### Step 2. Starting Databend
 
-1. Open a terminal and navigate to the folder where the extracted files and folders are stored.
+1. Configure an admin user. You will utilize this account to connect to Databend. For more information, see [Configuring Admin Users](../13-sql-clients/00-admin-users.md). For this example, uncomment the following lines to choose this account:
+
+```sql
+[[query.users]]
+name = "admin"
```

Comment:
In database, we usual use `root` rather than `admin` for the user name.

Reply:
Updated: 

[[query.users]]
name = "root"
auth_type = "no_password"

@b41sh Will you update the cfg file as well?